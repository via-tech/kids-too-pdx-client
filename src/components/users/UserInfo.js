import React from 'react';
import PropTypes from 'prop-types';

function UserInfo({ handleChange }) {
  return (
    <fieldset>
      <legend>User Info</legend>
      <label>Username or Email:
        <input type="text" name="username" onChange={handleChange} required />
      </label>

      <label>Password:
        <input type="password" name="password" onChange={handleChange} required />
      </label>
    </fieldset>
  );
}

UserInfo.propTypes = {
  handleChange: PropTypes.func
};

export default UserInfo;
