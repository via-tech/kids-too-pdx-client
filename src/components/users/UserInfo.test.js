import React from 'react';
import renderer from 'react-test-renderer';
import UserInfo from './UserInfo';

describe('User Info', () => {
  it('snapshots UserInfo', () => {
    const props = {
      handleChange: jest.fn()
    };

    const tree = renderer.create(
      <UserInfo {...props} />
    );

    expect(tree).toMatchSnapshot();
  });
});
