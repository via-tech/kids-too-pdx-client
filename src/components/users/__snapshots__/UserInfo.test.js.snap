// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Info snapshots UserInfo 1`] = `
<fieldset>
  <legend>
    User Info
  </legend>
  <label>
    Username or Email:
    <input
      name="username"
      onChange={[MockFunction]}
      required={true}
      type="text"
    />
  </label>
  <label>
    Password:
    <input
      name="password"
      onChange={[MockFunction]}
      required={true}
      type="password"
    />
  </label>
</fieldset>
`;
